apps/watch/src/components/ShareDialog/ShareDialog.spec.tsx

```diff
@@ -194,7 +198,10 @@ describe('ShareDialog', () => {
     })
 
     it('should share video to twitter', () => {
-      const facebookUrl = `https://twitter.com/intent/tweet?url=https://watch-jesusfilm.vercel.app/the-story-of-jesus-for-children`
+      mockRouter.push(
+        '/watch/the-story-of-jesus-for-children.html/english.html?r=0'
+      )
+      const facebookUrl = `https://twitter.com/intent/tweet?url=https://watch-jesusfilm.vercel.app/watch/the-story-of-jesus-for-children.html/english.html`
```

Comment:
```suggestion
      const twitterUrl = `https://twitter.com/intent/tweet?url=https://watch-jesusfilm.vercel.app/watch/the-story-of-jesus-for-children.html/english.html`
```

change to twitterUrl